src/zmq/zmqpublishnotifier.cpp

```diff
@@ -178,6 +180,16 @@ bool CZMQPublishHashTransactionNotifier::NotifyTransaction(const CTransaction &t
     return SendMessage(MSG_HASHTX, data, 32);
 }
 
+bool CZMQPublishHashTransactionEvictionNotifier::NotifyTransactionEviction(const CTransaction &transaction, MemPoolRemovalReason reason)
```

Comment:
Code between this function and `NotifyTransaction` is almost identical, could be de-dublicated, maybe also together with `NotifyBlock` but it's not blocking IMO

Reply:
Deduplicated the txn publishing portion, left block stuff alone for now.